html/changelogs/geeves-spacebike_storage.yml

```diff
@@ -0,0 +1,7 @@
+author: Geeves
+
+delete-after: True
+
+changes: 
+  - rscadd: "Spacebikes, speeders, and monowheels now have storage compartments accessible by dragging it onto your sprite."
+  - tweak: "Spacebikes and speeders now drive a bit faster on non-space tiles."
```

Comment:
If it leads to extensive usage inside of the station we will have to change that again.

Reply:
sounds good to me